### PR TITLE
[Echo3D] feature/echo 3d viewer

### PR DESCRIPTION
**Pull request by Noggling**
# Description

added echo 3d viewer

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

**Review by Andreas-Tonnessen — APPROVED** [review]

File: src/packages/ModelViewer/hooks/useLoadModel.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                setModel(model);
+                if (loadFullModel) {
+                    echo3DClient.viewer.fitCameraToModel(model);
+                }
+            })();
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [echo3DClient, currentPlant]);
+
+    useEffect(() => {
```
Comment (jaysencpp) [inline src/packages/ModelViewer/hooks/useLoadModel.ts:25]: Empty useeffect

File: src/packages/ModelViewer/ModelViewer.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+import { Button } from '@equinor/eds-core-react';
+import { tokens } from '@equinor/eds-tokens';
+import { useAppConfig, useAuthProvider, useFacility } from '@equinor/portal-client';
+import { useEffect, useRef } from 'react';
+import Icon from '../../components/Icon/Icon';
+import { ModelViewerContextProvider, useModelViewerContext } from './context/modelViewer';
+import { useModel } from './hooks/useLoadModel';
+import { Menu, Message, MessageWrapper, Wrapper, WrapperMenu } from './ModelViewerStyles';
+import { getModels, selectPlantByContext } from './utils/getCurrentContextModel';
+('');
```
Comment (jaysencpp) [inline src/packages/ModelViewer/ModelViewer.tsx:12]: what's this :D

Comment (Noggling) [reply src/packages/ModelViewer/ModelViewer.tsx:12]: :P

Comment (Noggling) [reply src/packages/ModelViewer/ModelViewer.tsx]: removed :P

**Review by jaysencpp — APPROVED** [review]

**Comment by Noggling** [comment]
merger den inn når vi har fåt ut test release

**Pull request merged**